tests/internals/fallback/fallback_test.go

```diff
@@ -196,6 +196,41 @@ spec:
       name: {{.TriggerAuthName}}
 `
 
+	scaledObjectTemplateWithoutMetricType = `
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: {{.ScaledObject}}
+  namespace: {{.Namespace}}
+  labels:
+    app: {{.DeploymentName}}
+spec:
+  scaleTargetRef:
+    name: {{.DeploymentName}}
+  minReplicaCount: {{.MinReplicas}}
+  maxReplicaCount: {{.MaxReplicas}}
+  fallback:
+    failureThreshold: 1
+    replicas: {{.DefaultFallback}}
+    behavior: currentReplicasIfHigher
```

Comment:
just making sure, `behavior` is an optional field, right?

Reply:
yep, it is :)
https://github.com/kedacore/keda/blob/a368eee4333c6328a351201b5b67fa0e0a0fb955/apis/keda/v1alpha1/scaledobject_types.go#L116-L119